docs/tutorials/how-to-estimator.ipynb

```diff
@@ -0,0 +1,174 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "35126242",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "\n",
+    "copyright:\n",
+    "  years: 2021, 2022\n",
+    "lastupdated: \"2022-03-21\"\n",
+    "\n",
+    "keywords: quantum, Qiskit, runtime, near time compute\n",
+    "\n",
+    "subcollection: quantum-computing\n",
+    "\n",
+    "---\n",
+    "\n",
+    "{{site.data.keyword.attribute-definition-list}}\n",
+    "\n",
+    "\n",
+    "# Estimator primitive\n",
+    "{: #program-estimator}\n",
+    "\n",
+    "\n",
+    "Understand the input and output for the Estimator primitive.\n",
+    "{: shortdesc}\n",
+    "\n",
+    "## Overview\n",
+    "{: #estimator-overview}\n",
+    "\n",
+    "The Estimator primitive returns an observable expectation value generated by given circuits executed on the target backend.  For a full, end-to-end example of how to use this primitive, see [Get started with Estimator](/docs/quantum-computing?topic=quantum-computing-example-estimator).\n",
```

Comment:
```suggestion
    "The Estimator primitive returns an observable expectation value generated by given circuits executed on the target backend.\n",
```
If we are going to have a "Get started with Estimator" page in this repo, then we can keep this line and fix the link, otherwise I'd just delete the line.